setup.sh

```diff
@@ -35,6 +35,7 @@ while true ; do
         --mipgaussian) MIPGAUSSIAN=true ; shift ;;
         --kaolin) KAOLIN=true ; shift ;;
         --nvdiffrast) NVDIFFRAST=true ; shift ;;
+        --gsplat) GSPLAT=true ; shift ;;
```

Comment:
To enable this option via this CLI, it needs to be added to the line `#2`, otherwise, an error saying `unrecognized option gsplat`.

Reply:
@Tradunsky 
Fixed, thanks for letting me know.